docs/Installation.md

```diff
@@ -61,3 +61,14 @@ Note that in order for this to work for pip versions >= 21.3, your Git version m
 ## As a Docker image
 
 Use one of the Dockerfiles in the [docker](../docker) directory to build an image with an environment already set up and ready for running NNCF [sample scripts](../README.md#model-compression-samples).
+
+## Corresponding versions
+
+The following table lists the recommended corresponding versions of backend packages
+as well as the supported versions of Python:
+
+| NNCF    | OpenVINO   | PyTorch             | ONNX     | ONNX Runtime   | TensorFlow   | Python           |
+| ------- | ---------- | ------------------- | -------- | -------------- | ------------ | ---------------- |
+| `2.6.0` | `2023.1.0` | `>=1.13.0`, `<2.1`  | `1.13.1` | `1.14.1`       | `2.12.0`     | `>=3.7`, `<3.11` |
```

Comment:
What backend versions do you want to provide? Do I mean backend versions that were used for testing or backend versions that were stated to be supported? In any case, please, align your table with Readme.txt.

Reply:
All versions get from setup.py
https://github.com/openvinotoolkit/nncf/blob/release_v260/setup.py
https://github.com/openvinotoolkit/nncf/blob/release_v250/setup.py
https://github.com/openvinotoolkit/nncf/blob/release_v240/setup.py

Reply:
Updated, get versions from README.md